src/main/scala/li/cil/oc/integration/projectred/EventHandlerProjectRed.scala

```diff
@@ -9,7 +9,7 @@ object EventHandlerProjectRed {
     player.getHeldItem.getItem match {
       case wrench: IScrewdriver =>
         if (changeDurability) {
-          wrench.damageScrewdriver(player.getEntityWorld, player)
+          wrench.damageScrewdriver(player, new ItemStack(player.getHeldItem.getItem))
```

Comment:
Would that actually damage held item though? I mean, why make new item stack here?

Reply:
The change was required because of some API missmatch. I am not sure if that is the right way to do it, but I did not see any other option. Do you?

Reply:
just pass player.getHeldItem ?